pandas/tests/io/parser/test_common.py

```diff
@@ -2079,3 +2080,17 @@ def test_integer_precision(all_parsers):
     result = parser.read_csv(StringIO(s), header=None)[4]
     expected = Series([4321583677327450765, 4321113141090630389], name=4)
     tm.assert_series_equal(result, expected)
+
+
+@td.skip_if_no("psutil")
+def test_file_descriptor_leak(all_parsers):
+    # GH 31488
+    import psutil
+
+    proc = psutil.Process()
```

Comment:
can you add a comment here very similar to what you posted above (e.g. why using psutil open files to check)

Reply:
added